lint-staged.config.js

```diff
@@ -10,6 +21,15 @@ module.exports = {
 	'iphone/TitaniumKit/TitaniumKit/Sources/API/TopTiModule.m': [
 		'npm run ios-sanity-check --'
 	],
-	'!(**/locales/**/*).js': 'eslint',
+	'*.js': async files => {
+		const filtered = await asyncFilter(files, async file => {
```

Comment:
Curious why this was added. Won't eslint ignore the passed in file if it's in the `.eslintignore` file anyways?